ariadne_django/formatters/errors/simple.py

```diff
@@ -0,0 +1,90 @@
+import django.core.exceptions
+
+from ariadne import get_error_extension
+
+from ariadne_django.formatters.errors.utils.extract_original_error import extract_original_error
+from ariadne_django.formatters.errors.utils.get_full_django_validation_error_details import (
+    get_full_django_validation_error_details,
+)
+
+
+ERROR_MAP = [
+    {
+        "classes": (django.core.exceptions.PermissionDenied,),
+        "code": "FORBIDDEN",
+        "message": "Access forbidden",
+        "details": {"non_field_errors": (["This request is understood, but is not able to be processed."])},
```

Comment:
It might be better to make some mention of authorization here. This error message doesn't make clear that permission was denied.